spec/integration_spec.rb

```diff
@@ -42,6 +42,14 @@ def compile_asset(name)
     expect(File.read(output_path)).to eq("<p>Component 1</p>\n")
   end
 
+  it 'can process complex HTML assets' do
+    compile_asset('complex1')
+    expect(File.read(output_path)).to eq(
+      "<a href=\"{{url}}\">Click</a>\n" +
+      "<select name=\"text\"><option value=\"value1\" selected=\"[[selected]]\">Value 1</option></select>\n"
```

Comment:
Can you use single-quoted strings and remove the escaping for these so its easier to read?